cccm-backend/cccm-api/src/main/resources/cccm-api.yaml

```diff
@@ -580,6 +582,70 @@ paths:
         404:
           description: Not Found
           content: {}
+  /forms/client/static99r:
```

Comment:
Taylor, could we change static99r to stat99r? stat99r is the type code for 'static 99r'. Trying to avoid another mapping in the frontend. :)